cvat/apps/consensus/apps.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2024 CVAT.ai Corporation
```

Comment:
Consider renaming the file into `merging_manager.py` or something similar to reflect it's contents better.